src/client/luafunctions.cpp

```diff
@@ -511,7 +511,6 @@ void Client::registerLuaFunctions()
     g_lua.registerClass<ThingType>();
     g_lua.bindClassStaticFunction<ThingType>("create", [] { return ThingTypePtr(new ThingType); });
     g_lua.bindClassMemberFunction<ThingType>("getId", &ThingType::getId);
-    g_lua.bindClassMemberFunction<ThingType>("getClothSlot", &ThingType::getClothSlot);
```

Comment:
This function you removed is being used here:
https://github.com/mehah/otclient/blob/bcefc963fa45498d02260008027332332c26dea3/modules/game_market/market.lua#L97

Reply:
It looked like a redundant copy, because it's also defined on line 533. I also noticed this function isn't working in the action bar.